### DISCUSSION TITLE
Hacktoberfest 2022 PR/MR excluded

### DISCUSSION THREAD

**Discussion by RickardAhlstedt** [Knackie/algorithmshacktoberfest2022#462] [General]
Make sure to check your hacktoberfest profile if your PR/MR got accepted to this repo, as my to PR's got excluded since the repo doesn't meet Hacktoberfest-values.

**Comment by Vishalc2901** [comment]
Plz add the hacktoberfest and hacktoberfest-accepted labels to this repo
